src/lib.rs

```diff
@@ -55,19 +74,19 @@ impl Client {
     /// Creates a new Client connected to a server or replica set using
     /// a MongoDB connection string URI as defined by
     /// [the manual](http://docs.mongodb.org/manual/reference/connection-string/).
-    pub fn with_uri(uri: &str) -> Result<Client> {
+    fn with_uri(uri: &str) -> Result<Client> {
         Client::with_uri_and_prefs(uri, None, None)
     }
 
     /// `with_uri` with custom read and write controls.
-    pub fn with_uri_and_prefs(uri: &str, read_pref: Option<ReadPreference>,
+    fn with_uri_and_prefs(uri: &str, read_pref: Option<ReadPreference>,
                               write_concern: Option<WriteConcern>) -> Result<Client> {
         let config = try!(connstring::parse(uri));
         Client::with_config(config, read_pref, write_concern)
     }
 
     fn with_config(config: ConnectionString, read_pref: Option<ReadPreference>,
-                   write_concern: Option<WriteConcern>) -> Result<Client> {
+        write_concern: Option<WriteConcern>) -> Result<Client> {
```

Comment:
I just went through all the files touched by this pull request again, so I think the indentation should be all set.